carbonmark/hooks/useFetchProject.ts

```diff
@@ -7,8 +8,9 @@ export const useFetchProject = (
   projectID: string,
   options?: SWRConfiguration
 ) => {
+  console.log("API.PROHJECTS", urls.api.projects);
```

Comment:
left a debug in here

Reply:
I'm glad it wasn't one of my "wtf" logs..

Removed